website/docs/docs/deploy/job-notifications.md

```diff
@@ -1,33 +1,83 @@
+    - To add an external email address, select the **Notification email** dropdown and choose **Add external email**. After you add the external email, it becomes available for selection in the **Notification email** dropdown list. External emails can be addresses that are outside of your dbt Cloud account and also for third-party integrations like [channels in Microsoft Teams](https://support.microsoft.com/en-us/office/tip-send-email-to-a-channel-2c17dbae-acdf-4209-a761-b463bdaaa4ca) and [PagerDuty email integration](https://support.pagerduty.com/docs/email-integration-guide).
 
-2. Select **Edit** to begin editing the **Email Notifications** settings.
-    - **As a user:** Choose the Notification type (Succeeds, Fails, or Is Cancelled) for each Job you want to receive notifications for.
+    <Lightbox src="/img/docs/deploy/example-notification-external-email.png" width="50%" title="Example of the Notification email dropdown"/>
 
-    - **As an admin:**  Under **Configure notifications for**, use the dropdown to select one or more users you'd like to set notifications for. If you only see your own name, then you might not have admin privileges. <br /><br />
-    Choose the Notification type (Succeeds, Fails, or Is Cancelled) for each Job you want them to receive notifications for.
+1. Select the **Environment** (in the dropdown) for the jobs you want to receive notifications about. 
 
-3. Click **Save**.
+1. Click **Edit** to configure the email notification settings. Choose one or more of the run statuses (**Succeeds**, **Fails**, **Is Canceled**) for each job you want to receive notifications about.
 
-    <Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/email-notifications.png" width="75%" title="Configuring Email Notifications"/>
+1. When you're done with the settings, click **Save**.
 
-### Slack
+    If there is another person you also want to send the notifications to, as an account admin, you can select another **Notification email** from the dropdown, **Edit** the job notification settings, and **Save** the changes.
+    
+    To set up alerts on jobs from a different environment, select another **Environment** from the dropdown, **Edit** those job notification settings, and **Save** the changes. 
 
-<Snippet path="slack-notifications-config-steps" />
+    <Lightbox src="/img/docs/deploy/example-email-notification-settings-page.png" width="100%" title="Example of the Email notifications page"/>
+
+### Unsubscribe from email notifications
+1. From the gear menu in the upper right corner, choose **Notification settings**.
+1. On the **Email notifications** page, click **Unsubscribe from all email notifications**. 
+
+## Slack notifications
+
+You can receive Slack alerts about jobs by setting up the Slack integration, then configuring the dbt Cloud Slack notification settings.
```

Comment:
Potentially save some space below. This may be better served as an individual page under setup as well
```suggestion
You can receive Slack notifications about jobs by setting up the [Slack integration](/faqs/Accounts/slack), then configuring the dbt Cloud Slack notification settings.
```